api/app/signals/apps/api/v1/views/signal.py

```diff
@@ -190,3 +190,16 @@ def children(self, request, pk=None):
 
         serializer = AbridgedChildSignalSerializer(child_qs, many=True, context=self.get_serializer_context())
         return Response(serializer.data)
+
+    def create(self, request, *args, **kwargs):
+        if isinstance(request.data, (list, )):
+            # We want some additional checks here
```

Comment:
Is this a TODO? Otherwise this comment is not very specific.

Reply:
Removed from the code

Reply:
I removed this from the code. These checks have been moved to the _SignalListSerializer.validate function